javaslang/src/main/java/javaslang/Value.java

```diff
@@ -109,16 +109,6 @@
 public interface Value<T> extends Iterable<T> {
 
     /**
-     * Shortcut for {@code exists(e -> Objects.equals(e, element))}, tests if the given {@code element} is contained.
-     *
-     * @param element An Object of type A, may be null.
-     * @return true, if element is contained, false otherwise.
-     */
-    default boolean contains(T element) {
-        return exists(e -> Objects.equals(e, element));
-    }
-
-    /**
```

Comment:
Yes, very good to move it to Traversable again. I did it, too, in a local branch.
The javadoc line `<li>{@link #contains(Object)}</li>` needs to be removed in class `Value`